CHANGELOG.md

```diff
@@ -67,6 +67,9 @@
 * Internal functions are now removed instead of invalidly imported if they are unused.
   [#4366](https://github.com/rustwasm/wasm-bindgen/pull/4366)
 
+* `web-sys` crate now fully works on `no_std`.
```

Comment:
```suggestion
* Fixed `no_std` support for all APIs in `web-sys`.
```
It sounds like the `std` `Vec` usage was on purpose, it was just an oversight :sweat_smile:!

Just for context: this is always a bug, because crate features have to be additive and they are not supposed to break compilation when you add a crate feature!